understanding/22/redundant-entry.html

```diff
@@ -35,11 +35,11 @@ <h2>Redundant Entry Success Criteria text</h2>
 
       <h2>Intent of Redundant Entry</h2>
 
-        <p>The intent of this Success Criterion is to ensure that users can successfully navigate multi-step processes. It reduces cognitive effort where  information is asked for more than once during steps in a process. It also reduces the need to recall information provided in a previous  step.</p>
+        <p>The intent of this Success Criterion is to ensure that users can successfully navigate multi-step processes. It reduces cognitive effort where  information is asked for more than once during a process. It also reduces the need to recall information provided in a previous  step.</p>
         
         <p>Information that is required to be remembered for input can pose a significant barrier to users with cognitive or memory difficulties. All users experience a natural gradual mental fatigue as they proceed through steps in a process. This fatigue is accelerated by the stress of recalling information from short-term working memory. Users with learning, and cognitive disabilities are highly susceptible to mental fatigue.</p>
         
-        <p>Requiring people to recall information already entered in the previous steps can cause them to give up or enter incorrect information. The autocomplete feature of browsers is not considered sufficient because it is the content (the web site) that needs to provide the stored information for a redundant entry, or avoid asking for the same information again.</p>
+        <p>Requiring people to recall information already entered in the previously can cause them to give up or enter incorrect information. The autocomplete feature of browsers is not considered sufficient because it is the content (the web site) that needs to provide the stored information for a redundant entry, or avoid asking for the same information again.</p>
```

Comment:
```suggestion
        <p>Requiring people to recall information previously entered can cause them to give up or enter incorrect information. The autocomplete feature of browsers is not considered sufficient because it is the content (the web site) that needs to provide the stored information for a redundant entry, or avoid asking for the same information again.</p>
```